rowan/functions.py

```diff
@@ -226,10 +226,10 @@ def power(q, n):
 
     Args:
         q ((..., 4) :class:`numpy.ndarray`): Array of quaternions.
-        n ((...) np.arrray): Scalars to exponentiate quaternions with.
+        n ((...) np.array): Scalars to exponentiate quaternions with.
```

Comment:
Didn't catch this in the first pass -- this is probably supposed to be:
```suggestion
        n ((...) :class:`numpy.ndarray`): Scalars to exponentiate quaternions with.
```